src/components/accounts/CreateAccount.js

```diff
@@ -25,6 +27,10 @@ class CreateAccount extends Component {
         }
     }
 
+    handleFundingKey = () => {
+        this.setState({fundingKey: new URLSearchParams(window.location.search).get('funding_key')});
```

Comment:
I think it might be better not to rely on `window.location` to make sure all works well with router:
https://learnwithparam.com/blog/how-to-handle-query-params-in-react-router/